lib/iris/tests/unit/representation/cube_printout/test_CubePrintout.py

```diff
@@ -537,10 +537,11 @@ def test_section_cell_methods(self):
         expected = [
             "name / (1)                          (-- : 1)",
             "    Cell methods:",
-            "        stdev                       area",
-            "        mean                        y (10m, vertical), time (3min, =duration)",
+            "        0                           area: stdev",
+            "        1                           y: time: mean: (interval: 10m"
+            " interval: 3min comment: vertical comment: =duration)",
         ]
-        self.assertEqual(rep, expected)
+        self.assertEqual(expected, rep)
```

Comment:
Why did you swap these?

Reply:
> Why did you swap these?

I believe this **_is_** better, but I only learnt it lately :  Because of how assertion routines are defined, and how it reports a failure, it should really be "expected-value, test-value".
Thus if x is "wrong-result", then assertEqual(x, 2) will report "2 != 'wrong' ", whereas assertEqual(2, x)  gives "'wrong-result' != 2".